examples/content/src/content/jcr_root/content/dam/core-components-examples/en/los-angeles/.content.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Need to reduce the number of assets we include for use in image / teaser / content fragments.